lang/sql/sql.py

```diff
@@ -0,0 +1,65 @@
+from talon import Context, Module, actions, settings
+
+mod = Module()
+ctx = Context()
+ctx.matches = r"""
+mode: user.sql
+mode: user.auto_lang
+and code.language: sql
+"""
+
+ctx.lists["user.code_functions"] = {
```

Comment:
Do you have any voice command to insert these functions? I had to do this myself (https://github.com/nriley/knausj_talon/blob/nriley/lang/sql/sql.py#L47)